### ISSUE TITLE
Digital Sequencer mk2 (poly) proposal

### ISSUE THREAD

**Issue by firolightfog** [clone45/voxglitch#112]
I see increasing interest in polyphonic modules and solutions. 

Have you ever considered to reshape your great Digital Sequencer with 
- polyphonic STEP input
- polyphonic RESET input 
- all sequence length (LEN) are moved to the context menu
- the old selection buttons are replaced with a big black TRACK SELECTOR knob 
- the old short keys are replaced and 1-9 would access relevant tracks
- polyphonic CV output
- polyphonic GATE output

![image](https://user-images.githubusercontent.com/34127628/149489042-94e632ac-849d-4c8a-a065-e11253d0385a.png)

**Comment by bret-2600** [comment]
I haven't given it as much thought as you, but the idea had crossed my mind!  What I'd probably do is create a new version called "Digital Sequencer XP"  (XP would mean xtra-special polyphonic.  Ha ha ha.)  I really like the idea of having 16 sequencers.  I prefer buttons to a track selector knob.  I think that they're faster to navigate.  But I could put two rows of 8, which would work great.  I like your idea of putting the sequence lengths in the context menu as well.  I'll get this on my to-do list!!

**Comment by bret-2600** [comment]
Here's a really early draft of the front panel.  It lacks any background coloring for the inputs and outputs, but you can see where I'm headed...
![image](https://user-images.githubusercontent.com/19557854/149634375-c4157455-2a62-4197-b7e4-ef0e88dd291d.png)

**Comment by S0210** [comment]
Very nice! Looking so simple but a lot of power under the hood. 

Let me have a GUI related question though. Is it a terrible amount of work to visually present the length of the active sequence by the number of columns? 

(Please don't take it as a suggestion. Just out of curiosity as I saw something similar elsewhere and I found it very handy.)

<img width="546" alt="image" src="https://user-images.githubusercontent.com/34127628/149637691-a92968a9-f3b5-4bc3-a207-06256161a8d7.png">

Best regards, Andras

**Comment by bret-2600** [comment]
Hello!  When I first was working on the sequencer, I played around with that idea but ended up not liking it as much visually.  So mostly for ascetic reasons I chose not to go in that direction.  One change that I should make is update the sequence length knobs to "snap" to discrete integers instead.  I think that would improve the feel and tooltips for those knobs.  :-)

**Comment by S0210** [comment]
Bello Hret,

`for ascetic reasons I chose not to go in that direction`
I can understand that.  

`sequence length knobs to "snap" to discrete integers `
I'd welcome that change for Digital Sequencer mkI. But it makes no sense for Digital Sequencer XP as there are no 'length knobs'. By the way I already identified the [module ](https://library.vcvrack.com/Bogaudio/Bogaudio-PolyCon) I'll be using with the LENGTH input CV of the coming Digital Sequencer XP. ;)

Looking forward to testing your wonderful modules.

Best regards, Andras
